packages/components/avatar/src/avatar.ts

```diff
@@ -10,80 +12,331 @@ export interface UserProfile {
+            />
+          </symbol>
+          <mask id="badge-cutout-${this.#avatarId}">
+            <rect width="${this.image.containerSize}" height="${this.image.containerSize}" fill="white" />
+            ${this.badgeCutout}
+          </mask>
+          <mask id="circle-${this.#avatarId}">
+            <rect
+              y="${this.image.y}"
+              x="${this.image.x}"
+              height="${this.image.size}"
+              width="${this.image.size}"
+              rx="${this.image.radius}"
+              fill="white"
+              mask="url(#badge-cutout-${this.#avatarId})"
+            />
+          </mask>
+        </defs>
+        <rect
+          y="${this.focusRingStrokeWidth / 2}"
+          x="${this.focusRingStrokeWidth / 2}"
+          height="${this.image.size + this.focusRingStrokeWidth + this.focusRingStrokeOffset * 2}"
+          width="${this.image.size + this.focusRingStrokeWidth + this.focusRingStrokeOffset * 2}"
+          rx="${this.image.radius + this.focusRingStrokeWidth / 2 + this.focusRingStrokeWidth}"
+          fill="transparent"
+          stroke-width="2"
+          stroke="var(--_focusring-color)"
+        />
+        ${this.badgeContent} ${this.imageContent}
+      </svg>
+    `;
+  }
+
+  override async connectedCallback(): Promise<void> {
+    super.connectedCallback();
+    await Config.getConfigSetting<AvatarConfig>('avatar').then(async config => {
+      this.avatarConfig = config;
+      this.borderWidth = this.avatarConfig?.badgeGapWidth * 2;
+      this.setAttribute('shape', this.avatarConfig.shape);
+      await this.#setBaseValues();
+    });
   }
 
   override render(): TemplateResult {
     return html`
-      <picture> ${this.image} ${this.statusBadge} </picture>
+      <picture> ${this.imageSVG} </picture>
       ${!this.imageOnly
         ? html`<div>
-            <span>${this.profileName}</span>
-            <slot></slot>
+            <span class="header">${this.profileName}</span>
+            <slot class="subheader"></slot>
           </div>`
         : nothing}
     `;
   }
 
-  override updated(changes: PropertyValues<this>): void {
+  override async updated(changes: PropertyValues<this>): Promise<void> {
     super.updated(changes);
 
+    if (changes.has('user')) {
+      this.errorLoadingImage = false;
+    }
+
     if (changes.has('orientation')) {
       /** it appears that converting the scss files removes this style property in the css, so i'm adding it here in a hacky way. */
```

Comment:
I thought i tried that and it didn't work, that's why i made the workaround.
But it should have been `value-no-vendor-prefix` because of course the value is prefixed, not the property :D